docs/pages/world/balance.mdx

```diff
@@ -183,13 +191,13 @@ uint256 balance = Balances.get(<namespace>);
 
    </CollapseCode>
 
-1. `pnpm dev` deploys the new code, but it deploys it to a new `World`.
-   Restart `pnpm dev` to have a `World` in the same address as before.
+1. `pnpm dev` deploys the new code, but it might deploy it to a new `World`.
+   Restart `pnpm dev` to ensure you have a `World` in the same address as before.
 
 1. Call `increment` with ETH.
 
    ```sh copy
-   cast send --private-key $PRIVATE_KEY 0x6E9474e9c83676B9A71133FF96Db43E7AA0a4342 --value 1ether "increment()"
+   cast send --private-key $PRIVATE_KEY $WORLD_ADDRESS --value 1ether "increment()"
```

Comment:
does it automatically read from `.env`? or do you need to run `source .env` first?

Reply:
Forge scripts read `.env` automatically. Cast commands do not.